app/src/main/java/com/kickstarter/libs/utils/UrlUtils.kt

```diff
@@ -0,0 +1,26 @@
+package com.kickstarter.libs.utils
+
+import android.content.Context
+import android.net.Uri
+import androidx.browser.customtabs.CustomTabsIntent
+import androidx.core.content.ContextCompat
+import com.kickstarter.R
+
+object UrlUtils {
+
+    fun baseCustomTabsIntent(context: Context): CustomTabsIntent {
```

Comment:
since we're just using the context to get the color, should we pass in the color?